### DISCUSSION TITLE
feat: The information "How the data is sent, verified, used" should be added

### DISCUSSION THREAD

**Discussion by arikan** [acikkaynak/deprem-yardim-frontend#831] [Ideas]
Absolutely to assure and answer basic questions about "Deprem yardım, afet bilgi, enkaz bildir, yakınımı bul" websites, those informations should take place clear and readable.

1. How the data is gathered, how much is there?
2. How the data is verified?
3. How the data is being used?

For example: depremyardim.com

1. How the data is gathered, how much is there?
From the forms and from twitter, instagram, whatsapp via scraping, filtering. How many are there in each address? https://afetharita.com

2. How the data is verified?
Unfortunately the is not a verification process (If there is: is the data is sent from different sources it is being marked.) If there is not any contact with the earthquake victim we can not mark yet, we will add if there is info from the area.

3. How the data is being used?
Data is being shared with the NGO's active in the area (another link? format?)
AFAD
AHBAP
AKUT
GEA
IHH
EGİAD

**Comment by eraygundogmus** [comment]
Note: PM is in confirmation process.

**Comment by eraygundogmus** [comment]
We are expecting UI design from pms in this topic @canerakin111

**Comment by Batuhandgenc** [comment]
I am working on UI

**Comment by 0xCakin** [comment]
@Batuhandgenc Has UX/UI been completed? If okey we can add in Progress.

**Comment by 0xCakin** [comment]
Burak will complete this tomorrow. If you can not get info, write to UI/UX leaders.

**Comment by gokceaydemir** [comment]
I can help with UI/UX

**Comment by alpermercan** [comment]
I think, writing each of those questions into faq cards with a little, readable style which redirects to a faq page which contains those questions could be good. Both the developers will be comfortable and if a new faq needs to be added there won't be any effort to edit each website. I can help with UI/UX if anyone has started designing. alpermerca#6121

**Comment by berkaydesign** [comment]
Design part is done. You can check it out on Figma as "Veri bilgisi pop-up". @arikan @canerakin111 
Content needs to be changed.
https://www.figma.com/file/sctw6xtcdoFOfmE0gC97Ft/Afetharita.com?node-id=2349%3A8093&t=v67k1CJDv8Ws7zE0-1

![image](https://user-images.githubusercontent.com/65855467/218256545-ff1761f5-60f1-4b0f-a8f4-b31821635ecb.png)

**Comment by berkaydesign** [comment]
The design is revised. We think that this information is not related directly to the end-user so we decided to remove the info icon and put the "Veri Bilgisi" next to "Gizlilik Politikası" as text. @arikan 

![image](https://user-images.githubusercontent.com/65855467/218261089-78e1dfa8-7a3f-462a-a1a7-342902427463.png)

**Comment by arikan** [comment]
@berkaydesign gayet iyi duruyor emeğine sağlık, bir iki not: 

- Veri kaynaklari: a b diye ayirmadan tek listede birleşirse daha basit olacak.

**Comment by berkaydesign** [comment]
![image](https://user-images.githubusercontent.com/65855467/218261256-cadd831b-dbc5-4595-871e-ffd82e2cd0ac.png)

**Comment by 0xCakin** [comment]
Let's keep talking on this. This must be completed but requires more massage. Moving this to Discussion.

**Comment by 0xCakin** [comment]
Listed in the backlog. Gathering more info. Will be included in the upcoming releases.
